go/pkg/tool/tool.go

```diff
@@ -848,3 +858,92 @@ func (c *Client) getActionResult(ctx context.Context, actionDigest string) (*rep
 	}
 	return resPb, nil
 }
+
+// IO is a collection input root Digest, Inputs, and Outputs for an action.
+type IO struct {
```

Comment:
naming suggestion: FlatActionIO

Wdyt about having the struct be even more direct? Like:

```
type FlatActionIO {
    RootDg string
    InputPaths map[string]string
    InputPathSymlinks map[string]string
    OutputFiles map[string]string
    OutputDirs map[string]string
    OutputFileSymlinks map[string]string
    OutputDirSymlinks map[string]string
}
```

Why do we need separate structs for Inputs and Outputs?